README.md

```diff
@@ -4,10 +4,33 @@ A github repository containing the **login-backend**, the **overworld** and all
 
 ## Structure
 
-| service        | port | address             |
-| -------------- | ---- | ------------------- |
-| login-backend  | 4000 | /api                |
-| login-frontend | 8000 | /                   |
-| overworld      | 1234 | /overworld          |
-| moorhuhn       | 7000 | /minigames/moorhuhn |
-| git-card-game  | 7001 | /minigames/git-card-game |
+| service           | address                      |
+| ------------------| -----------------------------|
+| login-frontend    | /                            |
+| login-backend     | /api/login/                  |
+| overworld         | /overworld                   |
+| moorhuhn          | /minigames/moorhuhn          |
+| git-card-game     | /minigames/git-card-game     |
+| crosswordpuzzle   | /minigames/crosswordpuzzle/  |
+
+## User manual
+
+Run the docker containers with the following command:
+```sh
+docker-compose up -d
```

Comment:
Since we are now using `latest` instead of referencing the exact versions (e.g. `v1.2.0`) of our images, this won't upgrade our containers. At least that's the default pull behavior of docker-compose.

https://docs.docker.com/compose/compose-file/#pull_policy

Reply:
In my opinion the production build on the server hast to be automatically updated to the latest release. For continuous integration we should serve always the latest releases and a release always should be production-ready.  

I plan also to set up a testing/development environment with containers from the main branch. But first I want to have a working production environment.

Reply:
@madebyTimo What you're mentioning is a basic point we should decide on early:
Is this repo intended to be a `prototype` for us to test combining all services, or is it also the application you have to download as a lecturer who wants to host our platform?
Because if it is the former, we can do your approach.
But if it is the latter, your approach is **strongly** discouraged: You don't want an existing application to suddenly break because you restarted it.

Reply:
> Since we are now using `latest` instead of referencing the exact versions (e.g. `v1.2.0`) of our images, this won't upgrade our containers. At least that's the default pull behavior of docker-compose.
> 
> https://docs.docker.com/compose/compose-file/#pull_policy

A took a bit research but it is hard to find a reliable answer to that question. But in my experience and from that what I have read docker pulls the image if a newer one exists. But if the container is already present docker doesn't updates the container but rather starts it and therefor didn't updated it. 

To update the containers I want to use [watchtower](https://containrrr.dev/watchtower/). Watchtower automatically recreates containers if a newer version exists. To automatically update to a newer version of the docker-compose file I want to implement a service to do a daily pull.

Reply:
> @madebyTimo What you're mentioning is a basic point we should decide on early: Is this repo intended to be a `prototype` for us to test combining all services, or is it also the application you have to download as a lecturer who wants to host our platform? Because if it is the former, we can do your approach. But if it is the latter, your approach is **strongly** discouraged: You don't want an existing application to suddenly break because you restarted it.
> 
> And my preference is towards being the complete binary.

@delvh This is no final run configuration to host it in a full production environment for open use. This is for our StuPro to have always a running environment with the actual status of  the project. Production is maybe the wrong name. It is the latest stable release from all services.